components/tidb_query/src/expr/ctx.rs

```diff
@@ -239,9 +239,17 @@ impl EvalContext {
         Err(err)
     }
 
+    pub fn handle_overflow(&mut self, is_overflow: bool) -> Result<()> {
```

Comment:
Looks like it's used nowhere. :/

Reply:
I need it in another PR. :laughing:

Reply:
I think you can combine them. This PR is not too big.